### DISCUSSION TITLE
lines jump, is it implemented ?

### DISCUSSION THREAD

**Discussion by tdenimal** [structurizr/ui#86] [General]
Hello, i was wondering if lines jump for crossing lines was supported  ? can't see anything (config or syntax) or anything in the doc. See examples of what i'm talking about with draw.io : https://drawio-app.com/blog/draw-io-now-supports-line-jumps/

if it does  not exists, it can be an interesting feature to add

**Comment by RowlandBanks** [comment]
There appears to be some movement towards this. The [v5.0.0 release](https://github.com/structurizr/java/releases/tag/v5.0.0) has the following note:

> structurizr-dsl: Adds support for a jump property on relationship styles.

So you can now do:

```
styles {
    relationship "Relationship" {
        jump true
    }
}
```

It doesn't have a visible effect currently as far as I can tell.

**Comment by simonbrowndotje** [comment]
Yes, line jumps were added in the last Lite/on-premises release (see the [Patreon post](https://www.patreon.com/posts/138426958) for more details).

<img width="1470" height="919" alt="Screenshot 2025-12-30 at 16 41 30" src="https://github.com/user-attachments/assets/7f429782-9521-4907-818e-1db1dcc348f5" />
